airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationRunner.java

```diff
@@ -0,0 +1,16 @@
+package io.airbyte.integrations.destination.snowflake;
+
+import io.airbyte.integrations.base.adaptive.AdaptiveDestinationRunner;
+
+public class SnowflakeDestinationRunner {
+
+
+  public static void main(final String[] args) throws Exception {
+    AdaptiveDestinationRunner.baseOnEnv()
+        .withOssDestination(() -> new SnowflakeDestination(OssCloudEnvVarConsts.AIRBYTE_OSS))
+        .withCloudDestination(() -> new SnowflakeDestination(OssCloudEnvVarConsts.AIRBYTE_CLOUD))
+        .run(args);
```

Comment:
this probably also needs to do a `SCHEDULED_EXECUTOR_SERVICE.shutdownNow();` similar to Akash's PR https://github.com/airbytehq/airbyte/pull/19314/files#diff-d6c8766f9065e3ddfc335cd448642c552b312f2d1feac08e62793aec3716df12R16

Reply:
updated, thanks!